src/plugins/data_source/server/client/configure_client.test.ts

```diff
@@ -13,16 +13,24 @@ import {
   SigV4Content,
 } from '../../common/data_sources/types';
 import { DataSourcePluginConfigType } from '../../config';
-import { ClientMock, parseClientOptionsMock } from './configure_client.test.mocks';
+import {
```

Comment:
Could you please check the CI tests before the merge? It appears that a couple of CI groups are failing, and the retry didn't go through.

Reply:
Yes. It is because of ChromeDriver installation failed. #5926 has been open to fix the same.